src/avs/SpeakDirectiveParser.js

```diff
@@ -63,14 +64,38 @@ export function extractAlexaTextResponses(alexaRawResponse) {
       );
     }
 
+    _validateDirective(avsDirective);
+    // Directives that are not Speak directives (for ex, ExpectSpeech** directive) will be skipped for now. If and when we
+    // start supporting voice, non-Speak directives should also be handled.
```

Comment:
Cant we handle ExpectSpeech directives by prompting user for more text rather than supporting voice? Isnt the project "Silent Alexa"? 
May be reword this comment and also create a feature request for future?

Reply:
Done.